### PR TITLE
Headline with level div.

### PR DESCRIPTION
**Pull request by apepper**
Allows to keep the styling of a heading, but remove it from the semantic hierarchy.

### CODE REVIEW

File: src/Widgets/HeadlineWidget/HeadlineWidgetClass.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const HeadlineWidget = provideWidgetClass('HeadlineWidget', {
   attributes: {
     alignment: ['enum', { values: ['left', 'center', 'right'] }],
     headline: 'string',
-    level: ['enum', { values: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] }],
+    level: ['enum', { values: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div'] }],
```
Comment (dcsaszar) [inline src/Widgets/HeadlineWidget/HeadlineWidgetClass.ts:7]: * [ ] FYI `level: "div"` doesn't make a lot of sense. Maybe a neutral value like `none`/`text` is a better fit?
* [ ] FYI Similarly in the editing config "div" is not really something a SEO editor cares about, maybe something like "Text only" or "No semantic tag" makes more sense.

File: src/assets/stylesheets/index.scss [hunk 2]
```diff
@@ -404,19 +404,6 @@ h6 {
   margin-bottom: 0.7rem;
 }
 
-div {
```
Comment (dcsaszar) [inline src/assets/stylesheets/index.scss:407]: * [ ] The styling of "Label headline" and "Label subtitle" changes when switching to the no-heading level. Changing the SEO level should never change the chosen style.
image
image

**Review by dcsaszar — CHANGES_REQUESTED** [review]

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (apepper) [reply src/assets/stylesheets/index.scss:407]: I think this is an independent issues with `Label heading` and `Label subtitle` that already existed prior to this PR. I think it should be addressed separately, once our designer is available again.

Comment (dcsaszar) [reply src/assets/stylesheets/index.scss:407]: I'm confused. We introduce `div` headlines in this PR. How is it even possible for a `div` headline related issue to have existed before that?

Comment (apepper) [reply src/assets/stylesheets/index.scss:407]: `Label heading` and `Label subtitle` will already (prior to this PR) automatically select `<div>` if no heading tag is set.

Comment (dcsaszar) [reply src/assets/stylesheets/index.scss:407]: I see, it's the "unset" case. Thanks for clarifying!

**Pull request merged**